build.yaml

```diff
@@ -169,6 +162,105 @@ common_cli:
       fi
     }
 
+    # Retries a command if it fails, using the exponential backoff algorithm.
+    # Usage:
+    # retry_n_times_with_timeout <max retries> <timeout in seconds> command
+    #
+    function retry_n_times_with_timeout {
+      set +e
+      max_tries=\${1}
+      seconds=\${2}
+      shift 2
+
+      let delay=15
```

Comment:
I mean that:

 1. `delay` specifically should have a comment saying that it's in seconds
 2. Maybe `delay` should be a parameter, e.g. 

```
retry_n_times_with_timeout <max retries> <timeout in seconds>  <initial delay in seconds>
```